### PR TITLE
[ready] fixes/support for multilevel clinic demo

### PR DESCRIPTION
**Pull request by ddengster**
~~depends on https://github.com/osrf/rmf_demos/pull/136~~ merged!

~~waiting for #80~~ merged!

- crash fix
- negotiation trajectories now show up on multilevel

run `ros2 launch demos clinic.launch.xml` instead of the office demo

### CODE REVIEW

File: src/robot-trajectory-manager.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export interface Trajectory {
   segments: RawKnot[];
   robot_name: string;
   fleet_name: string;
+  map_name: string;
```
Comment (matiasbavera) [inline src/robot-trajectory-manager.ts:41]: Just to check is the `map_name` === Level name, right? 
Also if the `map_name` property is required you may want to add it to `trajectories.json` too. For the 
`npm run start:mock` to work

Comment (ddengster) [reply src/robot-trajectory-manager.ts:41]: add map_name to json 6910e26

File: src/components/negotiations-panel.tsx [hunk 2]
```diff
@@ -58,8 +58,7 @@ export interface NegotiationsPanelProps {
   negotiationStatusUpdateTS: number; // used to trigger rerenders
 }
 
-
-export const NegotiationsPanel = React.memo((props: NegotiationsPanelProps) => {
+export default function NegotiationsPanel(props: NegotiationsPanelProps): JSX.Element {
```
Comment (matiasbavera) [inline src/components/negotiations-panel.tsx:61]: Maybe I'm missing something or maybe is a rebase mistake. But you're removing the memo here (which improves performance). 

You can check the difference on `src/components/tests/__snapshots__/negotiations-panel.test.tsx.snap` too

Related PR https://github.com/osrf/romi-dashboard/pull/112

**Review by matiasbavera — APPROVED** [review]
After that issue it's resolved, you should be able to merge it.

File: src/components/negotiations-panel.tsx (hunk 2 above)
Comment (ddengster) [reply src/components/negotiations-panel.tsx]: nice catch cd53210

**Pull request merged**